specs/latest/2.0/index.html

```diff
@@ -1202,13 +1202,15 @@ <h2 class="no-toc">Table of contents</h2>
               an <code>INVALID_VALUE</code> error is generated.
           <li>If <code>srcByteOffset</code> is less than zero, an <code>INVALID_VALUE</code>
               error is generated.
-          <li>If zero is bound to <code>target</code>, an <code>INVALID_OPERATION</code>
+          <li>If no WebGLBuffer is bound to <code>target</code>, an <code>INVALID_OPERATION</code>
               error is generated.
           <li>If <code>target</code> is <code>TRANSFORM_FEEDBACK_BUFFER</code>,
               and any transform feedback object is currently active, an <code>INVALID_OPERATION</code>
               error is generated.
           <li>If <code>srcByteOffset + length*ElementSize(dstBuffer)</code> is larger
               than the length of the GL buffer, generate INVALID_OPERATION.
+          <li>If <code>dstBuffer.length</code> equals <code>dstOffset</code>, no data is written to
+            <code>dstBuffer</code>.
```

Comment:
Is the implication that length == 0 in this case since this check is listed after "if dstOffset+length is greater than dstBuffer.length"? That's not obvious. It would be possible for the user to specify both the dstOffset and length arguments, instead of taking the default computed value, and then both the first and last rules in this list could apply.

Should this instead state something like: "if the computed length equals zero, no data is written to dstBuffer"?